layouts/partials/logo.html

```diff
@@ -1,3 +1,3 @@
 <a id="logo" href="/">
-  <img src="{{ .Site.BaseURL }}/images/sakuli_01.svg" width="90%" height="auto" />
+  <img src="{{ .Site.BaseURL }}/images/sakuli_01.svg" width="250px" display="block" height="auto" />
```

Comment:
```suggestion
  <img src="{{ .Site.BaseURL }}/images/sakuli_01.svg" style="width: 250px; display: block; height: auto" />
```

Please test this suggestion. The attributes you applied had no effect since e.g. an `<img>` tag doesn't support a `display` property. This is a CSS Attribute that has to be set via `style` attribute or a CSS rule. Anyway: An `img` tag  is displayed as a block element by default.